src/pudl/settings.py

```diff
@@ -428,7 +428,7 @@ def make_datasources_table(self, ds: Datastore) -> pd.DataFrame:
         datasets_in_datastore_format = {
             name: setting
             for (name, setting) in datasets_settings.items()
-            if name in ds.get_known_datasets()
+            if name in ds.get_known_datasets() and setting is not None
```

Comment:
Can you explain the relevance of this change to this PR, @zaneselvans ? It's not immediately clear to me whether this is fixing a little bug you came across during the PR.

Reply:
I was trying to run the ETL without any EPA CEMS data in my settings file, but then the validated settings object had `None` for the EPA CEMS settings, which caused the construction of this little metadata table to fail. Somehow it had just never been tested.

This change restricts the datasets listed in the little metadata table to include only that that had non-null settings. I.e. the ones that were actually used in the construction of the DB.